### DISCUSSION TITLE
Road blockers of helix from being a daily driver for me

### DISCUSSION THREAD

**Discussion by ta3pks** [helix-editor/helix#8255] [General]
I'm not sure if this is the right place to write this. I apologize if it is not. However, I tried the Matrix channel to initiate some communication, but it seems like the majority of folks don't actually use the Matrix channel (or perhaps I just visited during a quiet hour or something).

I really love Helix. I use Neovim for work, and though the way of editing things is different, I was able to get the hang of it and become somewhat efficient in a relatively short amount of time.

I tried using Helix as my daily driver at work instead of Neovim for two whole days. I really, really wanted to make the switch, and for the most part, it is possible. However, there are only two issues that make it impossible for me to use Helix, aside from some hobby stuff. The first is the lack of plugin authoribility, and the second is the inability to alter configuration items for certain filetypes (similar to ftplugins in Vim).

Most other things are minor, and I believe they could be simply solved by the community through plugins. Once the community can author plugins, the floodgates would be opened for Helix.

Another issue (although it wouldn't be a problem if plugin authoring were as simple as in Vim) is having a proper language for configuration.

Thanks for this amazing project. I really love Helix, and I genuinely want the Helix project to be successful.

**Comment by pascalkuthe** [comment]
Issues (just like in NY opensource project) are not the right place for this. Both of your concerns already have open issues where the development ks tracked.
